Document zabbix publish_string_states config (#36734)
home-assistant/home-assistant.github.io: source/_integrations/zabbix.markdown

  description: The host that will receive the state changes from Home Assistant. It needs to be manually created in Zabbix first and have the template associated with it (see above).
  required: false
  type: string
publish_string_states:

>   description: Also publish string states, i.e. states which cannot be cast to a numeric value.

Reviewer: ```suggestion
  description: Also publish string states. That is, states that cannot be cast to a numeric value.
```

Author: It was already merged - @c0ffeeca7 should I re-open the PR?
